notebooks/BikeShareExample.ipynb

```diff
@@ -622,14 +622,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "forecastResponse = forecastquery.get_forecast(\n",
+    " forecastResponse = forecastquery.get_forecast(\n",
     "    PredictorName=predictorName,\n",
     "    Interval=\"hour\",\n",
-    "    Filters={\"item_id\":\"bike_12\"}\n",
+    "   Filters={\"item_id\":\"bike_12\"}\n",
     ")\n",
     "print(forecastResponse)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Generate Plot \n",
+    "print(\"Forcast for P90\") \n",
+    "d = pd.DataFrame.from_dict(forecastResponse['Forecast'])\n",
+    "df = pd.DataFrame.from_dict(d.loc['mean']['Predictions']).dropna().rename(columns = {'Val':'p90'})\n",
```

Comment:
@lv6520  You take the `mean` column but you label it as `P90`. It's a typo right?

Reply:
yes .. fixed